src/t_hash.c

```diff
@@ -1178,7 +1178,7 @@ void hrandfieldWithCountCommand(client *c, long l, int withvalues) {
     }
 }
 
-/* HRANDFIELD [<count> WITHVALUES] */
+/* HRANDFIELD key [count [WITHVALUES]] */
```

Comment:
```suggestion
/* HRANDFIELD key [<count> [WITHVALUES]] */
```
i don't think that's right. `<count>` denotes a variable, and `[COUNT]` denotes an optional keyword.